packages/prisma/schema.prisma

```diff
@@ -244,9 +244,13 @@ model EventType {
 
   bookingRequiresAuthentication Boolean @default(false)
 
+  // Inbound agents for this event type
+  inboundAgents Agent[] @relation("inboundAgentEventType")
```

Comment:
As we're getting more selective about putting relationships in only when absolutely necessary, I see no access patterns like ```select: { ..., inboundAgents: { select } }``` - so I'm wondering if we need this relationship or whether we can leave this; as it's not important for Cascade.